### DISCUSSION TITLE
how to disable anti knockback????

### DISCUSSION THREAD

**Discussion by Marmok228aboba121371624bobanikita666ban** [ORelio/Minecraft-Console-Client#2763] [Q&A]
help me, I do not know how to disable the anti knockback function. and is it possible to do this?

**Comment by milutinke** [comment]
The MCC doesn't support physics, thus it doesn't support knockback. There is nothing you can do about it.

**Reply by Marmok228aboba121371624bobanikita666ban** [reply]
OK, send me the documentation and I'll write him this property myself.

**Reply by milutinke** [reply]
> OK, send me the documentation and I'll write him this property myself.

It's not as simple as you think, if it was, we would add it a long time ago.
The code of the client is very old, and it requires a refactoring or a re-write to support physics, because it's a mess of spaghetti code

**Reply by Marmok228aboba121371624bobanikita666ban** [reply]
Sorry, I didn't take that into account. I will hope that it will be updated soon.
